main-packages-list.txt

```diff
@@ -8,6 +8,7 @@ mod_ssl
 procps
 python3-jinja2
 python3-mod_wsgi
+python3-setuptools
```

Comment:
this should not be needed for all installations, only source installations
let's move it inside the BUILD_DEPS in prepare-image.sh at L24

Reply:
done